pyaedt_with_IDE.bat

```diff
@@ -54,7 +55,7 @@ for %%c in (%env_vars%) do (
         set version=!env_var_name:ANSYSEM_ROOT=!
         set versions[!choice_index!]=!version!
         set version_pretty=20!version:~0,2! R!version:~2,1!
-        echo [!choice_index!]!version_pretty!
+        echo [!choice_index!] !version_pretty!
```

Comment:
I'm not sure what "verions_pretty" is but all tech guide lines say to avoid the use of exclamation points.

Reply:
@PipKat  that is the way variables are referenced inside a .bat file. Can't change that.

Reply:
@kbhagat-ansys Thank for explaining the "why" because it helps me learn what can and can't be changed!